src/elements/swapLimit/SwapLimitTable.tsx

```diff
@@ -39,115 +41,163 @@ export const SwapLimitTable = () => {
+                src={payToken.logoURI}
+                alt="Token"
+                className="bg-grey-2 rounded-full h-28 w-28 mr-5"
+              />
+              {`${payToken.symbol} ${payAmount.value}`}
+            </div>
+          );
+        },
+        minWidth: 200,
+        sortDescFirst: true,
+      },
+      {
+        id: 'getAmount',
+        Header: 'You get',
+        accessor: 'getAmount',
+        Cell: (getAmount) => {
+          const getToken = getAmount.row.original.getToken;
+          return (
+            <div className={'flex items-center'}>
+              <Image
+                src={getToken.logoURI}
+                alt="Token"
+                className="bg-grey-2 rounded-full h-28 w-28 mr-5"
+              />
+              {`${getToken.symbol} ${getAmount.value}`}
+            </div>
+          );
+        },
+        minWidth: 200,
+        sortDescFirst: true,
+      },
+      {
+        id: 'rate',
+        Header: 'Rate',
+        accessor: 'rate',
+        Cell: (rate) => rate.value,
+        minWidth: 200,
+        sortDescFirst: true,
+      },
+      {
+        id: 'filled',
+        Header: 'Filled',
+        accessor: 'filled',
+        Cell: (filled) => prettifyNumber(filled.value ?? 0) + '%',
+        minWidth: 70,
+        sortDescFirst: true,
+      },
+      {
+        id: 'actions',
+        Header: '',
+        accessor: 'hash',
+        Cell: (cellData) => {
+          return (
+            <button
+              className={'hover:text-error py-5 pl-5 transition duration-200'}
+              onClick={async () =>
+                dispatch(
+                  addNotification(
+                    await cancelOrders([cellData.row.original.orderRes])
+                  )
+                )
+              }
+            >
+              <IconTimes className={'w-10'} />
+            </button>
+          );
+        },
+        width: 50,
+        minWidth: 50,
+        disableSortBy: true,
+      },
+    ],
+    [dispatch]
+  );
+
   if (!account || orders.length === 0) return null;
 
   return (
-    <div className="md:rounded-30 bg-white dark:bg-blue-4 md:shadow-widget my-40 pb-10">
-      <div className="flex flex-col md:flex-row md:justify-between md:items-center md:h-60 md:px-20">
-        <div>
-          <h2>Active Orders</h2>
-        </div>
-        <div
-          className={'flex flex-col md:flex-row md:items-center md:space-x-10'}
-        >
+    <section className="content-section pt-20 pb-10 mt-20">
+      <div className="flex justify-between items-center mb-20 mx-[20px] md:mx-[44px]">
+        <h2>Active Orders</h2>
+        <div className="flex items-center gap-10">
           <SearchInput
             value={search}
             setValue={setSearch}
             className="max-w-[160px] rounded-10 h-[35px]"
           />
-          <div className={'flex'}>
+          <button
+            className={'btn-outline-secondary btn-sm rounded-10'}
+            onClick={async () =>
```

Comment:
In general, please prefer a handler function wrapped by useCallback - and not an inline arrow function